prow/plank/controller.go

```diff
@@ -441,3 +459,13 @@ func (c *Controller) getBuildID(name string) (string, error) {
 	}
 	return "", err
 }
+
+func pendingJobs(pjs []kube.ProwJob) map[string][]kube.ProwJob {
+	rjm := map[string][]kube.ProwJob{}
```

Comment:
I think we prefer `rjm := make(map[string][]kube.ProwJob)`, although I'm not sure about that.